src/main/java/eu/xenit/gradle/docker/compose/DockerComposeConventionImpl.java

```diff
@@ -108,23 +124,37 @@ public void fromProject(Project project) {
                 composeTask.doFirst(CONFIGURE_COMPOSE_ACTION_NAME, new Action<Task>() { // No lambda -> see #96
                     @Override
                     public void execute(Task t) {
-                        createSetComposeEnvironmentFromPathAction().execute(dockerBuildImage);
+                        if(environmentVariable != null) {
+                            String addition = "_" + dockerBuildImage.getName();
+                            if(dockerBuildImage.getName() == "buildDockerImage") {
```

Comment:
Always add the full environment variable as well, for consistency.

For the sorter name, this is handled below: https://github.com/xenit-eu/alfresco-docker-gradle-plugin/pull/184/files#diff-759745ef3412ddc0e8adc15dc325e986d85c8bffc25ab61e09bbc00e842b54e5R144-R145